pkg/operator/ceph/cluster/osd/osd.go

```diff
@@ -264,6 +272,9 @@ func (c *Cluster) startProvisioningOverPVCs(config *provisionConfig) {
 			schedulerName:    volume.SchedulerName,
 			encrypted:        volume.Encrypted,
 			deviceSetName:    volume.Name,
+			encryption: encryptionSpec{
```

Comment:
Why not just add kmsconfig directly on the struct? 
```suggestion
			KMSConfig: volume.EncryptionKMSConfig,
```

Reply:
I was thinking we might have more in the future.